python/lsst/ts/ATWhiteLightSource/wlsSimComponent.py

```diff
@@ -5,7 +5,7 @@
 
 
 class Client:
-    def connect():
+    def connect(self):
```

Comment:
Please document what this class is for. I am guessing that the connect method emulates something in the modbus library, but I'm not clear why.

If it does emulate a 3rd party library class then clearly this `connect` should match the library, so I won't suggest making this one asynchronous.

Reply:
Yeah, this is a band-aid fix. I've actually made the all the modbus stuff asynchronous in another CSC, and written a simulator that puts test signals on the modbus analog input channels , and that should be easy to adapt for this. But all my modbus hardware is in transit to Chile right now and i want to make sure the async updates work with the actual hardware before I mess with it here, where what i have has actually been tested with the real hardware.

Reply:
Sounds wise.